railties/lib/rails/generators/rails/scaffold_controller/templates/api_controller.rb

```diff
@@ -41,6 +41,7 @@ def update
   # DELETE <%= route_url %>/1
   def destroy
     @<%= orm_instance.destroy %>
+    render json: {}, status: :no_content
```

Comment:
I remember that one of the tests was failing when I originally pushed, but I think that's because I changed the default to render a 406 if no content was provided. Removed the line and pushed to Travis, let's see how it turns up!